docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: /dev/getting-started/paper-plugins
+---
+
+# Paper Plugins
+
+Paper plugins allow developers to take advantage of more modern concepts introduced by Mojang, such as datapacks, to 
+expand the field of what the Paper API is able to introduce.
+
+:::info Experimental
+
+This is experimental and may be subject to change.
+
+:::
+
+- [Bootstrapper](#bootstrapper)
+- [Loader](#loaders)
+- [Differences](#differences)
+
+## How do I use them?
+Similarly to Bukkit plugins, you have to introduce a ``paper-plugin.yml`` file into your jar resources folder.
```

Comment:
To me personally (and the reason I started this review), this sentence doesn't make it super obvious if this replaces the `plugin.yml` or is needed as a supplement. I imagine if the plugin loads on Paper and Bukkit it should include both, but that should be made obvious to allow the developer to decide what platforms they wish to support.